dictionaries/html-symbol-entities/cspell-ext.json

```diff
@@ -7,7 +7,7 @@
     "dictionaryDefinitions": [
         {
             "name": "html-symbol-entities",
-            "file": "./entities.txt.gz",
```

Comment:
`entities.txt.gz` is the correct filename. It is automatically generated during the publishing process.